src/HolographicCamera.Unity/ProjectSettings/ProjectSettings.asset

```diff
@@ -0,0 +1,678 @@
+%YAML 1.1
+%TAG !u! tag:unity3d.com,2011:
+--- !u!129 &1
+PlayerSettings:
+  m_ObjectHideFlags: 0
+  serializedVersion: 15
+  productGUID: 604586f275d10a745b4adb9e37216dc1
+  AndroidProfiler: 0
+  AndroidFilterTouchesWhenObscured: 0
+  AndroidEnableSustainedPerformanceMode: 0
+  defaultScreenOrientation: 4
+  targetDevice: 2
+  useOnDemandResources: 0
+  accelerometerFrequency: 60
+  companyName: Microsoft
+  productName: HolographicCamera
```

Comment:
>HolographicCamera [](start = 15, length = 17)

nit: I don't have a strong preference on this choice, but it may be worth including something like SpectatorView.HolographicCamera in this app name. Otherwise if you are in the process of uninstalling applications it may be hard to remember what this application was for.